Tests/PostgRESTTests/PostgrestBuilderTests.swift

```diff
@@ -1,23 +0,0 @@
-//
-//  PostgrestBuilderTests.swift
-//  Supabase
-//
-//  Created by Guilherme Souza on 20/08/24.
-//
-
-@testable import PostgREST
-import XCTest
-
-final class PostgrestBuilderTests: XCTestCase {
```

Comment:
Why was this test removed?

Reply:
Because This test has only Custom Header Test.
Shouldn't we remove?

Reply:
This actually tests that when you set custom headers on a builderB that was derived from a builderA, builderA doesn't get affected.

We should keep the test.